src/handlers/ExecuteViewSubmitHandler.ts

```diff
@@ -479,20 +487,20 @@ export class ExecuteViewSubmitHandler {
 
         const createdRecord = await NotionSdk.createRecord(
             access_token,
-            database,
+            newDatabase,
             data
         );
 
         let message: string;
 
         if (createdRecord instanceof Error) {
             this.app.getLogger().error(createdRecord.message);
-            message = `🚫 Something went wrong while creating record in **${workspace_name}**.`;
+            message = `Something went wrong while creating record in **${workspace_name}**.`;
             await sendNotification(this.read, this.modify, user, room, {
                 message,
             });
         } else {
-            const { info } = database;
+            const { info } = newDatabase;
             const databasename = info.name;
```

Comment:
Here, as i mentioned above.
```suggestion
            const databasename = info.name.replace("📚","");
```